### PR TITLE
Add Temper II and changes Multi Strike effect

### PR DESCRIPTION
**Pull request by AppleCronch**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Adds Temper II as a working spell, changes multi strike effect to buff double/triple attack depending on the spell.

## Steps to test these changes

Use temper II and it'll buff triple attack instead of double attack from temper

### CODE REVIEW

**Review by WinterSolstice8 — APPROVED** [review]

File: sql/spell_list.sql [hunk 1]
```diff
@@ -860,7 +860,7 @@ INSERT INTO `spell_list` VALUES (880,'drain_iii',0x00000000000000630000000000000
 -- INSERT INTO `spell_list` VALUES (892,'luminohelix_ii',0x00000000000000000000000000000000000000000000,8,7,0,1,0,0,2000,48000,0,0,288,1000,0,0,1.00,0,0,0,0,NULL);
 -- INSERT INTO `spell_list` VALUES (893,'full_cure',0x00000000000000000000000000000000000000000000,8,7,0,1,0,0,2000,48000,0,0,288,1000,0,0,1.00,0,0,0,0,NULL);
 -- INSERT INTO `spell_list` VALUES (894,'refresh_iii',0x00000000000000000000000000000000000000000000,8,7,0,1,0,0,2000,48000,0,0,288,1012,0,0,1.00,0,0,0,0,NULL);
--- INSERT INTO `spell_list` VALUES (895,'temper_ii',0x00000000000000000000000000000000000000000000,8,7,0,1,0,0,2000,48000,0,0,288,1000,0,0,1.00,0,0,0,0,NULL);
+INSERT INTO `spell_list` VALUES (895,'temper_ii',0x00000000630000000000000000000000000000000000,6,142,7,0,1,34,72,7000,36000,0,0,120,2000,0,0,1.00,1,300,0,0,'SOA');
```
Comment (Xaver-DaRed) [inline sql/spell_list.sql]: Where did you get this data from, if I may ask?

Concretelly, the spell animation.

Comment (AppleCronch) [reply sql/spell_list.sql]: Oh I just copied the one from temper and changed the values. Never considered that the animation could be different

Comment (WinterSolstice8) [reply sql/spell_list.sql]: image
it's 43

Comment (WinterSolstice8) [reply sql/spell_list.sql]: Whoops, wrong packet. 1011.
image

Comment (AppleCronch) [reply sql/spell_list.sql]: okay fixed

**Review by Xaver-DaRed — APPROVED** [review]

**Pull request merged**